reference/fleet/install-elastic-agents.md

```diff
@@ -109,9 +109,13 @@ For containerized environments, the basic {{agent}} flavor is installed using th
 
 For containerized environments, the servers {{agent}} flavor is installed using the default `elastic-agent` command with an agent container package. This flavor contains the same set of components described in [Servers flavor](#elastic-agent-servers-flavor-regular) above.
 
+{applies_to}`stack: ga 9.1` The servers {{agent}} flavor also includes the [journald](https://www.freedesktop.org/software/systemd/man/latest/systemd-journald.service.html) dependencies necessary to use the [journald input](beats://reference/filebeat/filebeat-input-journald.md).
```

Comment:
@colleenmcginnis, is this placement up-to-date with the latest versioning guidelines?

Reply:
We haven't (yet) decided to standardize the placement of inline labels. This is perfect for now.

(I'll be doing a sweep of all 9.1 labels across ingest docs the week before the 9.1 release to make any final updates so as long as the label exists and I can find it later, I'm happy.)